services/static-webserver/client/source/class/osparc/Application.js

```diff
@@ -318,6 +318,8 @@ qx.Class.define("osparc.Application", {
         case "s4l":
         case "s4llite":
         case "s4lacad":
+        case "s4ldesktop":
```

Comment:
Should you start adding these in https://github.com/ITISFoundation/osparc-simcore/blob/23dad29fc1b98a8891e4a6b0b90a4931a10201df/services/web/server/src/simcore_service_webserver/statics/_constants.py#L3

Reply:
Just getting ready once it's there